guardian/prosemirror-typerighter
# chore: Bump dependencies and tidy up

## What does this change?

A housekeeping PR in anticipation of #208:

- bumps `prosemirror-utils` to avoid a dependency on `prosemirror-tables`
- bumps `typescript` and `node` to latest
- reorganises `node_modules` to ensure dependencies and dev dependencies are correctly sorted
- refactors a few CSS operations to avoid warnings
- speeds up the build a bit with `skipLibCheck` (but not much ... it's still 11s!)

## How to test

The commits should be well-structured, so reviewing commit by commit may make it easier if there's anything that is unclear from the diff. Happy to split this PR further if the number of changes makes this difficult to review.

Steps to test:

- The automated tests should pass, and the project should build (this is covered by CI)
- Run the project locally with `npm run watch`. It should run in the browser as expected.
